### DISCUSSION TITLE
Get rid of blurriness on image sprite when zoomed out?

### DISCUSSION THREAD

**Discussion by Donny-H** [pixijs/pixijs#9249] [Q&A]
I am loading a simple sprite in pixijs like so:

```
         let img = new PIXI.Sprite.from(bgSrc);
         img.width = 1191;
         img.height = 842;
         img.interactive = true;
```

bgSrc is a png image which serves as the background of the main canvas. The problem is that when zoomed out, the details of the image get extremely blurry, pixelated, hard to see. Especially because the png contains text on it which is required to be read. When zooming in onto the png, the clarity becomes much better and the text can be read. I understand that this is due to the way pixi scales the image at different zoom levels, but how can this be fixed? I need the image clarity to be good at a further zoom level. I have tried many things like setting PIXI.settings.SCALE_MODE = PIXI.SCALE_MODES.NEAREST; before any rendering (as well as setting this for the sprite itself with img.texture.baseTexture.scaleMode) but nothing seems to make an appreciable difference.

Any help is appreciated, thanks.

**Comment by SuperSodaSea** [comment]
Maybe you can try to set `renderer.resolution = window.devicePixelRatio; renderer.resize(renderer.width, renderer.height);` when `devicePixelRatio` changed? (Probably you also need to set `autoDensity: true` option when creating `PIXI.Application`.)
